### PR TITLE
ui: remove dead clusterviz code

### PR DESCRIPTION
**Pull request by couchand**
Remove dead/unused code from the clusterviz folder.

### CODE REVIEW

File: pkg/ui/ccl/src/views/clusterviz/containers/map/modalLocalities.tsx [hunk 1]
```diff
@@ -83,107 +70,42 @@ class NodeBox extends React.Component<NodeBoxProps, any> {
...
-
-  toggleMode = () => {
-    const mode = this.state.mode === "tree" ? "map" : "tree";
-    this.setState({ mode });
-  }
-
   render() {
     const localities = new LocalityCollection();
     this.props.nodeHistories.forEach(nh => localities.addNode(nh));
     localities.computeTreeMapLayout(this.props.zoom.viewportSize());
```
Comment (vilterp) [inline pkg/ui/ccl/src/views/clusterviz/containers/map/modalLocalities.tsx]: looks like this call should be removed if we're removing the TreeMap. Also the method `computeTreeMapLayout` should be removed.

**Review by vilterp — APPROVED** [review]
Noted one more bit of dead code, otherwise good 👍

File: pkg/ui/ccl/src/views/clusterviz/containers/map/modalLocalities.tsx (hunk 1 above)
Comment (couchand) [reply pkg/ui/ccl/src/views/clusterviz/containers/map/modalLocalities.tsx]: Yep, there's also a handful of comments that I noticed I missed, too.

Comment (couchand) [reply pkg/ui/ccl/src/views/clusterviz/containers/map/modalLocalities.tsx]: Oh nvm, those comments were left over from my previous pr, not this one.

**Pull request merged**